apps/design-tokens-ui/src/app/components/nav/nav.component.scss

```diff
@@ -38,8 +53,12 @@ $logo-height: 28px;
       fill: var(--color-neutral-50);
     }
 
-    .design-tokens-ui-nav-logo img {
-      filter: brightness(0);
+    .design-tokens-ui-nav-logo {
+      width: 100%;
+
+      img {
+        filter: brightness(0);
```

Comment:
I'm pretty sure there is a version of the logo out there, that we do not need to recolor with css-filters, but it's fine for now.